core/src/lib.rs

```diff
@@ -129,8 +129,6 @@
 
 // Make sure all our public APIs have docs.
 #![warn(missing_docs)]
-// Deny unused qualifications.
-#![deny(unused_qualifications)]
```

Comment:
This is because fuzz test failed to compile. quite weird. Anyway I don't think there's need to use the `deny` level.. `warn` is enough.

<img width="1266" alt="image" src="https://github.com/apache/opendal/assets/37948597/47fd4035-c71b-4ae9-8408-a643996be9f7">